lib/pf/web/dispatcher.pm

```diff
@@ -43,6 +43,9 @@ use pf::Portal::Session;
 use pf::web::externalportal;
 use pf::inline;
 
+use pfconfig::cached_scalar;
+tie our $portal_request_timeout, 'pfconfig::cached_scalar', 'resource::portal_request_timeout';
```

Comment:
pf::config will be loaded by all the controller and model in catalyst so it really does not change that much.